### PR TITLE
[feature/336342: Do you hold a waste permit exemption?]

### PR DESCRIPTION
**Pull request by shehzad-ismail8**

### CODE REVIEW

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                saveButton,
+                PermitExemptionResources.ErrorMessage))
+            {
+                return View(viewModel);
+            }
+
+            await _wastePermitService.UpdatePermitExemption(viewModel);
+
+            if (saveButton == SaveButton.SaveAndContinue && viewModel.HasPermitExemption.Value == true)
+            {
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Remove the braces if there's only one line after an if

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                return RedirectToAction("ExemptionReferences", "Accreditation");
+            }
+            else if (saveButton == SaveButton.SaveAndContinue && viewModel.HasPermitExemption.Value == false)
+            {
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: As above

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                return RedirectToAction("AuthorityToIssues", "Accreditation");
+            }
+            else
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Don't really need the else here. The other 2 if statements do a return, so if the code gets after that point, then it is by definition the "else"

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Services/Accreditation/WastePermitService.cs [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+            return new PermitExemptionViewModel
+            {
+                Id = id,
+                HasPermitExemption = await _httpWastePermitService.GetHasPermitExemption(id)
+            };
+        }
+
+        public async Task UpdatePermitExemption(PermitExemptionViewModel permitExemptionViewModel)
+        {
+
```
Comment (NikPeters-Atos) [inline src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Services/Accreditation/WastePermitService.cs]: blank line

**Review by NikPeters-Atos — CHANGES_REQUESTED** [review]

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs (hunk 1 above)
Comment (shehzad-ismail8) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Removed

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs (hunk 2 above)
Comment (shehzad-ismail8) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Removed

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs (hunk 3 above)
Comment (shehzad-ismail8) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Controllers/AccreditationController.cs]: Removed else

File: src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Services/Accreditation/WastePermitService.cs (hunk 4 above)
Comment (shehzad-ismail8) [reply src/EPR.Accreditation.Portal/EPR.Accreditation.Portal/Services/Accreditation/WastePermitService.cs]: Removed

**Review by NikPeters-Atos — APPROVED** [review]

**Pull request merged**